src/main/java/com/training/web/pages/rccl/HomePage.java

```diff
@@ -0,0 +1,49 @@
+package com.training.web.pages.rccl;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class HomePage{
+    public SelenideElement popUpTermsandCondition = $x("//button[normalize-space(text()) = 'Accept']");
+    public SelenideElement headerName = $x("//h2[contains(text(),'audrey')]");
+    public SelenideElement popUpPrivacyPolicy= $x("//button[contains(text(),'Accept')]");
+    public SelenideElement  clickCruiseButton= $x("//button[contains(text(),'Add a cruise')]");
```

Comment:
extra space between SelenideElement & clickCruiseButton.
Also the element name should be cruiseButton and not the action which you are performing